scipy/stats/_kde.py

```diff
@@ -364,7 +364,7 @@ def integrate_box_1d(self, low, high):
                         special.ndtr(normalized_low)))
         return value
 
-    def integrate_box(self, low_bounds, high_bounds, maxpts=None):
+    def integrate_box(self, low_bounds, high_bounds, maxpts=None, rng=None):
```

Comment:
If were adding a new argument maybe we should consider making in kwarg only?

Reply:
Good point, included in gh-22624.